src/main/charts/bitbucket/values.yaml

```diff
@@ -1553,3 +1553,59 @@ openshift:
   # configuration files as ConfigMaps.
   #
   runWithRestrictedSCC: false
+
+# OpenSearch config
+#
+opensearch:
+
+  # -- Deploy OpenSearch Helm chart and automatically
+  # configure Bitbucket to use it as a search platform
+  #
+  enabled: false
```

Comment:
How does upgrade work? When I upgrade bitbucket helm charts and something has changed on the opensearch definition as well, what will happen?